host/xdc/elasticsearch_test.go

```diff
@@ -18,8 +18,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-// +build !race
-// +build esintegration
+//go:build !race && esintegration
```

Comment:
These file was modified after I ran `make bins`, not sure if it will break something. Same with the next modification

Reply:
Did you try “make fly”?

Reply:
I specifically made this new changes in a new commit just in case I need to revert :) I ran `make fmt` and I got `make: Nothing to be done for 'fmt'`. I reverted the last commit (removing all this changes) and tried `make fmt` and got the same message. Should I revert them manually?

Reply:
Seems like a go 1.17 thing: https://golang.org/doc/go1.17#build-lines

Since we're not officially on 1.17... tbh I'm not sure if we should do this or not.  And I'm fairly surprised that `go fmt` does this since our stated version is 1.13, and older versions of go will not maintain them - if we ever change these build conditions and are *not* running <1.17, go fmt will ignore the new format, and 1.17+ will use the old conditions because it's now preferred.

Reply:
Are you referring to https://github.com/uber/cadence/blob/38881a8f996ce3230a77b7acc946438fbc532895/Dockerfile#L7 

I think we should change it to 1.16. 
@noiarek can you try using 1.16? If you use homebrew, there is a command to switch the go version. 

Thanks @Groxx !

Reply:
I meant https://github.com/uber/cadence/blob/master/go.mod#L3 (actually go 1.12 apparently), but that works too / I imagine we build the tests from that dockerfile?  If so, it wouldn't detect out-of-sync format directives without upgrading, yeah.

Reply:
@Groxx  I've installed go 1.16.7 and ran `make fmt` after it and I'm still getting this modifications. Any advice?